src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+You can also have a look at the
+[source code on Glitch](https://glitch.com/edit/#!/handwriting-recognition).
+
+> TODO: Glitch takes ages to start…
+
+## Security and permissions
+
+The Chromium team has designed and implemented the Handwriting Recognition API using the core
+principles defined in [Controlling Access to Powerful Web Platform Features][powerful-apis],
+including user control, transparency, and ergonomics.
+
+### User control
+
+The Handwriting Recognition API is currently implemented behind a flag. If the flag is enabled, the
+API is available by default and can't be turned off by the user. It is only available for websites
+delivered via HTTPS.
+
+### Transparency
+
+At the time of this writing, there is no indication that the API is in active use.
+
+### Permission persistence
+
+The Handwriting Recognition API currently does not show any permissions prompts. Thus, permission
+does not need to be persisted in any way.
+
+## Feedback
+
+The Chromium team wants to hear about your experiences with the Handwriting Recognition API.
+
+### Tell us about the API design
+
+Is there something about the API that doesn't work like you expected? Or are there missing methods
+or properties that you need to implement your idea? Have a question or comment on the security
+model? File a spec issue on the corresponding [GitHub repo][issues], or add your thoughts to an
+existing issue.
+
+### Report a problem with the implementation
+
+Did you find a bug with Chromium's implementation? Or is the implementation different from the spec?
+File a bug at [new.crbug.com](https://new.crbug.com). Be sure to include as much detail as you can,
+simple instructions for reproducing, and enter `Blink>Handwriting` in the **Components** box.
+[Glitch](https://glitch.com/) works great for sharing quick and easy repros.
+
+### Show support for the API
+
+Are you planning to use the Handwriting Recognition API? Your public support helps the Chromium team
+prioritize features and shows other browser vendors how critical it is to support them.
+
+Share how you plan to use it on the [WICG Discourse thread][wicg-discourse]. Send a tweet to
+[@ChromiumDev][cr-dev-twitter] using the hashtag
+[`#HandwritingRecognition`](https://twitter.com/search?q=%23HandwritingRecognition&src=recent_search_click&f=live)
+and let us know where and how you're using it.
+
+## Helpful Links
+
+- [Explainer][explainer]
+- [GitHub repo][github]
+- [ChromeStatus](https://www.chromestatus.com/features/5263213807534080)
+- [Chromium bug](https://crbug.com/1166910)
+- [TAG review](https://github.com/w3ctag/design-reviews/issues/591)
+- [Intent to Prototype][i2p]
+
+> TODO: CrBug not public yet
+
+## Acknowledgements
+
+This article was reviewed by [Joe Medley]. Thanks to [Thinktecture](https://www.thinktecture.com)
+for providing the demo implementation. Hero image by [Samir Bouaked](https://unsplash.com/@sbouaked)
```

Comment:
```suggestion
This article was reviewed by [Joe Medley]. Hero image by [Samir Bouaked](https://unsplash.com/@sbouaked)
```

I fear this won't work. No sponsorship is allowed.

Reply:
Alternative proposal: Add the company more prominently to your bio. Right now you have http://localhost:8080/authors/christianliebel/.